client/checkout/woopay/express-button/woopay-express-checkout-button.js

```diff
@@ -69,7 +87,81 @@ export const WoopayExpressCheckoutButton = ( {
 		}
 	}, [ isPreview, context ] );
 
-	const initWooPay = ( e ) => {
+	const newIframe = useCallback( () => {
+		const getWoopayOtpUrl = () => {
+			const tracksUserId = JSON.stringify(
+				getConfig( 'tracksUserIdentity' )
+			);
+
+			const urlParams = new URLSearchParams();
+			urlParams.append( 'testMode', getConfig( 'testMode' ) );
+			urlParams.append( 'source_url', window.location.href );
+			urlParams.append( 'tracksUserIdentity', tracksUserId );
+
+			return getConfig( 'woopayHost' ) + '/otp/?' + urlParams.toString();
+		};
+
+		const addEventListeners = ( iframe ) => {
+			iframe.addEventListener( 'load', () => {
+				sessionDataPromise.then( ( response ) => {
+					iframe.contentWindow.postMessage(
+						{
+							action: 'setPreemptiveSessionData',
+							value: response,
+						},
+						getConfig( 'woopayHost' )
+					);
+				} );
+			} );
+		};
+
+		const iframe = document.createElement( 'iframe' );
+		iframe.src = getWoopayOtpUrl();
```

Comment:
Temporarily using the `/otp` URL to establish communication between the merchant site and WooPay site.